tests/ethpm/test_package_ipfs_initialization.py

```diff
@@ -0,0 +1,27 @@
+import pytest
+
+from ethpm import Package
+
+
+VALID_IPFS_PKG = 'ipfs://QmeD2s7KaBUoGYTP1eutHBmBkMMMoycdfiyGMx2DKrWXyV'
+
+
+def test_package_from_ipfs_with_valid_uri():
+    package = Package.from_ipfs(VALID_IPFS_PKG)
```

Comment:
It appears this is actually making the http request.  This is generally considered an anti-pattern.  Instead, I'd recommend using the `monkeypatch` pytest fixture to mock out the `fetch_ipfs_package` function with a canned response.